controllers/common/util.go

```diff
@@ -37,13 +37,6 @@ func SetObjectMeta(owner, obj metav1.Object, gvk schema.GroupVersionKind) error
 		obj.SetNamespace(owner.GetNamespace())
 	}
 
-	objLabels := obj.GetLabels()
```

Comment:
The added `owner-id` is not used by anything, but it changes the passed in object metadata labels (here, the passed in objects are `Sensor`, `EventSource` and `EventBus`), as a result, it causes some unexpected issues, such as the `Service` `labelSelector` generated by `EventSource` doesn't match the POD's labels - This is side effect of PR #773. 

Simply remove this to fix the issue. To avoid issues when upgrading (removing a label from EventBus StatefulSet is not allowed), the `owner-id` label need to be added to the passed in objects as below.